scripts/linters/js_ts_linter.py

```diff
@@ -1139,6 +1139,109 @@ def _check_comments(self):
+            if 'implements' in class_name:
+                class_name = class_name.split('implements')[0]
+            if '<' in class_name:
+                class_name = class_name.split('<')[0]
+            return class_name
+
+        angular_services_index_path = (
+            './core/templates/services/angular-services.index.ts')
+        injectables_to_ignore = [
+            'ExplorationStats',
+            'InteractionAttributesExtractorService',
+            'NormalizeWhitespacePunctuationAndCasePipe\n',
+            'MockCsrfTokenService',
+            'StatePropertyService',
+            'StateNextContentIdIndexService\n',
+            'SubtitledUnicodeObjectFactory',
+            'UpgradedServices'
+        ]
+        if self.verbose_mode_enabled:
+            python_utils.PRINT('Starting Angular Services Index file check')
+            python_utils.PRINT('----------------------------------------')
+        angular_services_index = self.file_cache.read(
+            angular_services_index_path)
+        summary_messages = []
+        total_error_count = 0
+        total_files_checked = 0
+        for file_path in self.ts_files:
+            file_content = self.file_cache.read(file_path)
+            if '@Injectable({' in file_content:
+                total_files_checked += 1
+                class_name = get_injectable_class_name(file_content)
+                if class_name in injectables_to_ignore:
+                    continue
+                import_statement = 'import { %s' % class_name
+                if not import_statement in angular_services_index:
+                    summary_message = (
+                        'Please import %s to Angular Services Index file in %s'
+                        % (class_name, angular_services_index_path))
+                    summary_messages.append(summary_message)
+                    python_utils.PRINT(summary_message)
+                    python_utils.PRINT('')
+                    total_error_count += 1
+
+                service_name_type_pair = (
+                    '[\'%s\', %s]' % (class_name, class_name))
+
+                if not service_name_type_pair in angular_services_index:
+                    summary_message = (
+                        'Please add the pair %s to the angularServices in %s'
+                        % (service_name_type_pair, angular_services_index_path)
+                    )
+                    summary_messages.append(summary_message)
+                    python_utils.PRINT(summary_message)
+                    python_utils.PRINT('')
+                    total_error_count += 1
+        with linter_utils.redirect_stdout(sys.stdout):
+            if self.verbose_mode_enabled:
+                python_utils.PRINT('----------------------------------------')
+            if total_error_count:
+                python_utils.PRINT('(%s files checked, %s errors found)' % (
+                    total_files_checked, total_error_count))
```

Comment:
Can you please share a screenshot of the lint fail log?

Reply:
![image](https://user-images.githubusercontent.com/23002114/92202123-e6ef3780-ee9b-11ea-8b3c-9a37501ef317.png)